src/index.html

```diff
@@ -8,22 +8,8 @@
 
     <title page-title>Bitwarden Web Vault</title>
 
-    <script>
-        (function () {
-            // Set theme on page load
-            // This is done outside the Angular app to avoid a flash of unthemed content before it loads
-            let theme = window.localStorage.getItem('theme');
-            if (theme?.indexOf('system') > -1) {
-                theme = window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light';
-            } else if (theme?.indexOf('dark') > -1) {
-                theme = 'dark';
-            }
-            else {
-                theme = 'light';
-            }
-            document.documentElement.classList.add('theme_' + theme);
-        })();
-    </script>
+    <script src="scripts/theme.js" crossorigin="anonymous" type="application/javascript"
```

Comment:
We should add the cache buster.

Reply:
We could also replace this logic with https://github.com/architgarg/html-webpack-injector but it might overcompensate things.

Reply:
I wasn't _really_ happy with how I'd integrated this with our build process, so I revisited bundling it using Webpack. In some ways it makes it more complex, but I think it's better to have everything in the same build process. Here's what I've done:

* added `babel-loader` and dependencies to `devDependencies`. These are the latest versions.
* use `babel-loader` instead of `file-loader` to process `theme.js`
* move `theme.js` out of the `scripts` dir so it doesn't get caught by the `CopyWebpackPlugin` which is pointing at this dir
* use `html-webpack-injector` to move `theme.js` into the `<head>` tags. This is a very small and not particularly active repo, but it does one thing and does it in a fairly transparent manner. The alternative would be to manually place our `<script>` tags in the template, which is doable but messier and harder to maintain. Because this is a less reputable source, I've locked it to the current package version (no `~` or `^` in the version number).
* we don't have `integrity` hashes on our webpack bundles, so this is no longer an issue (can be revisited later if desired)
* webpack handles cache busting for us

Webpack is still a learning curve for me, so please tell me if this can be improved. Also let me know if you have any concerns about introducing these dependencies.